City-of-Helsinki/drupal-asuntotuotanto
# ASU-245 React implementation

How to test: 
- `make fresh`
- Go to https://asuntotuotanto.docker.sh/fi/asuntohaku and check that the react application gets loaded
